statvar_imports/usa_dol/usa_dol_wages/usa_dol_wages_download_script.py

```
@@ -53,6 +55,7 @@ def extract_all_table_data(url):
     except requests.exceptions.RequestException as e:
```

> print(f"No tables found on the page {url}")
Should we throw an error if no data could be found in html, instead of silently ignoring?